Makefile

```diff
@@ -36,6 +36,7 @@ parser-test:
 TINYGO_STACKS:=-stack-size=40mb
 
 wasm: Makefile *.go */*.go $(GEN) wasm/wasm_exec.js wasm/wasm_exec.html wasm/grol_wasm.html
+	sed -i 's/console.warn("scheduleTimeoutEvent: missed timeout event");/this._scheduledTimeouts.delete(id);\nconsole.warn("scheduleTimeoutEvent: missed timeout event");/'  wasm/wasm_exec.js
```

Comment:
just do the sed in a pipe instead of the copy (where wasm/wasm_exec.js is being created/copied a bit below)